identity/src/main/java/com/android/identity/Util.java

```diff
@@ -29,6 +29,7 @@
 import androidx.annotation.Nullable;
 
 import androidx.annotation.VisibleForTesting;
+import androidx.core.util.Preconditions;
```

Comment:
Do we need this import?

Reply:
No, removed.

I'll push --force, since your comments on this commit were very minor and I've resolved both of them.